s4e-web/cypress/integration/map/map-locations-search.spec.ts

```diff
@@ -21,85 +21,75 @@ context('Map Locations Search', () => {
       .clear();
   });
 
+  function waitForCitiesList() {
+    cy.server()
+    cy.route('GET', '/api/v1/places?namePrefix=warsz').as('getCityList');
+    cy.wait('@getCityList').its('status').should('eq', 200)
```

Comment:
I'm not sure if it suppose to be placed in here

Reply:
I can't put it after cy.visit() because '@getCityLits' is waiting for list of city which are loading after type name and click button search or enter

Reply:
I put function in PO